test/index.js

```diff
@@ -20,6 +22,20 @@ const walletTwoAddresses = [
   '0xbbafcf3d00fb625b65bb1497c94bf42c1f4b3e78',
   '0x49dd2653f38f75d40fdbd51e83b9c9724c87f7eb',
 ];
+
+// Shims for globals that don't exist in test environment
+global.crypto = {
+  getRandomValues: () => {
+    return Math.floor(Math.random(1000000));
+  },
+  subtle: {
+    digest: (algorithm, data) =>
+      new Uint8Array(Date.now().toString().split('')).buffer,
+  },
```

Comment:
At present, a few of the tests are being effected by the fact that I believe `TextEncoder.decode` is working improperly in the tests.  Each time it appears to return an empty string, which it shouldn't be.  So either the shim is off or the result of what I'm shimming `digest` to do is wrong.